apps/rmh-web/app/pt-portal/page.tsx

```diff
@@ -15,9 +17,9 @@ export default function PTPortal() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <TopBar />
-      <main className='flex min-h-screen justify-center w-full'>
+      <main className="flex min-h-screen w-full">
+        <SideBar isLoading={isLoading} users={users} />
```

Comment:
might make sense to just move the useUsers hook into sidebar instead of passing it down